fa_IR.ini

```diff
@@ -1,91 +1,93 @@
 # //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 # Don't Delete This, Please!
+# Current Translation by: Leopard20
+# Original Translator: آراس رضایی
```

Comment:
No need to add these lines, this is already addressed by the "translators**n** =" lines.